File: test/fixtures/php-guzzle/j_patch_array.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->patch('http://localhost:28139/patch', [
    'headers' => [
        'Content-Type' => 'application/x-www-form-urlencoded'
    ],
    'body' => 'item[]=1&item[]=2&item[]=3'
]);

File: test/fixtures/java-unirest/j_patch_array.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.patch("http://localhost:28139/patch")
            .header("Content-Type", "application/x-www-form-urlencoded")
            .body("item[]=1&item[]=2&item[]=3")
            .asString();
    }
}
